doc/data_structures.rst

```diff
@@ -2,22 +2,24 @@
 Data Structures
 ***************
 
-RedisAI defines three new data structures to be
```

Comment:
You could move those quick summaries of the client and dataset to the top of this file and then go into more detail below

Reply:
Hmmmm I could but then the client API definition would seem a little out of place I feel, what are your thoughts on me describing what a client is? somewhere else in the documentation... like under the smartredis tab?

Reply:
but also this tab is discussing the data structures we offer and not about the API's

Reply:
but it def wouldnt hurt to have the api defs at the top here tbh - i can adapt it to be like data structures work with which api and if they r in mem or not - r these ds sent to the db or something like that, thoughts on that?

Reply:
Hmm. It sounds like what we really need is an API overview page that talks about what the individual pieces of the API are and then links to the pages for them. I can help you set that up